src/traits/drop.md

```diff
@@ -34,7 +34,7 @@ fn main() {
 Discussion points:
 
 * Why doesn't `Drop::drop` take `self`?
-    * Short-answer: If it did, `drop` would be called at the end of
+    * Short-answer: If it did, `std::mem::drop` would be called at the end of
```

Comment:
```suggestion
    * Short-answer: If it did, `drop` (`std::mem::drop`) would be called at the end of
```

Reply:
Actually, reading this again, it's not really true: `std::mem::drop` wouldn't be called, there's nothing special about it. The value would just be dropped.

Reply:
Well it's not called directly, it's called as part of dropping the value.